File: client/src/app/shared/models/phone.ts
import { User } from './user';
import { PhoneExpense } from './phone-expense';

export class Phone {
    constructor(public id?: string,
                public type?: string,
                public ddd?: string,
                public number?: string,
                public dateCreated?: string,
                public dateUpdated?: string,
                public enabled?: boolean,
                public phoneExpenses?: PhoneExpense[],
                public user?: User) {
    }
}

File: api/Models/Phone.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Cretovale_api.Models
{
  public class Phone : IEntity
  {
    public Guid Id { get; set; }

    [Required]
    public string Type { get; set; }

    [Required]
    public string DDD { get; set; }

    [Required]
    public string Number { get; set; }

    public bool Enabled { get; set; }

    public User User { get; set; }
    public ICollection<PhoneExpense> PhoneExpenses { get; set; }

    public DateTime DateCreated { get; set; }
    public DateTime DateUpdated { get; set; }
  }
}
